Make adjustments to the notebook based on feedback

felipepaes opened the pull request:
Hey @esloch! This PR adds more context about why we chose to use Random Forest for the regression task and also provides clarification regarding the choice of total notifications as the study's dependent variable.

This commit addresses specific feedback points as follows:
  - Clarifies the use of Random Forest for regression task
  - Explains why we chose to use Random Forest
  - Reinforces the choice of total notifications as dependent variable

Please let me know your thoughts on the changes and/or any suggestions for adjustments needed.

felipepaes committed:
Make adjustments to the notebook based on feedback

This commit addresses specific feedback points as follows:
  - Clarifies the use of Random Forest for regression task
  - Explains why we chose to use Random Forest
  - Reinforces the choice of total notifications as dependent variable

esloch commented on notebooks/NB_FINAL.ipynb:
```diff
@@ -1110,13 +1110,15 @@
+    "\n",
+    "Ao enfrentar o desafio de prever o número total de casos notificados (`casos_notif`), optamos por utilizar o algoritmo Random Forest, uma escolha amplamente reconhecida em problemas de regressão. Este algoritmo é especialmente eficaz quando a variável a ser prevista é contínua, como é o caso da nossa aplicação, em que buscamos predizer o valor numérico representando o total de casos notifica ...
+    "\n",
+    "O Random Forest é um algoritmo mplamente utilizado tanto para tarefas de regressão quanto para classificação. Entretanto, pelo fato de a variável que desejamos predizer com o algoritmo ser contínua (valor numérico representando o total de casos notificados), utilizamos o Random Forest para executar a tarefa de regressão. Sua capacidade de combinar várias árvores de decisão resulta em previsõ ...
```
The explanation is somewhat redundant; please change to:
 
"Ao abordar a previsão do número total de casos notificados (casos_notif), escolhemos o algoritmo Random Forest, amplamente reconhecido para problemas de regressão. Este algoritmo é especialmente eficaz para variáveis contínuas, como no nosso caso, onde buscamos prever o valor numérico que representa o total de casos notificados. A escolha do Random Forest para esta tarefa se baseia em sua capacidade comprovada de combinar várias árvores de decisão, resultando em previsões robustas e precisas para variáveis contínuas."

esloch commented on notebooks/NB_FINAL.ipynb:
```diff
@@ -1126,7 +1128,7 @@
     "#### Particionando os Dados\n",
     "\n",
-    "Aqui é onde selecionaremos os **atributos preditore**(variáveis independentess), aqueles que serão utilizados para encontrar nossso atributso-alvo (variável dependente), ou seja, o número total de notificações de incidências de dengue. Particionaremos os dados com a função `train_test_split` da biblioteca sklearn na proporção **80% para treino** e **20% para teste**, respectivamente.\n"
+    "Aqui é onde selecionaremos os **atributos preditores** (variáveis independentes), aqueles que serão utilizados para encontrar nossso atributo-alvo (variável dependente), ou seja, o número total de notificações de incidências de dengue. Particionaremos os dados com a função `train_test_split` da biblioteca sklearn na proporção **80% para treino** e **20% para teste**, respectivamente.\n"
```
"o número absoluto de casos notificados."
Change to:
"...ou seja, o número absoluto de casos notificados de dengue para o Estado do Ceará."

esloch reviewed (commented):
Thank you for work on this.

felipepaes commented:
@esloch, thank you for your feedback! I have implemented the requested adjustments. Additionally, I have added a note to clarify that RF can be used for classification tasks as well, as mentioned by professor Flávio in his feedback.

Please, let me know if there are any changes needed 🙏

esloch reviewed (approved):
LGTM!
